app/javascript/app/styles/layout.scss

```diff
@@ -10,7 +10,7 @@
 
 %grid {
   display: grid;
-  grid-column-gap: $gutter-padding;
+  grid-column-gap: $grid-gap;
```

Comment:
Why does this need changing? This is universal to the whole app?

Reply:
Isn't this change for the whole app?

Reply:
But i dont understand why you have changed to grid-gap instead of just changing the value of gutter-padding